### ISSUE TITLE
Vídeo ?

### ISSUE THREAD

**Issue by DevJoseWeb** [cleuton/FaceGuard#1]
Cleuton boa noite, vai ter um vídeo do FaceGuard - Face Recognition + Machine Learning + IoT Demo rodando ?

**Comment by cleuton** [comment]
Sim. O código definitivo ainda não está no repositório. Falta a parte do
Raspi. Devo filmar no finfi e postar segunda.

Em 31 de jan de 2018 9:55 PM, "Junior" <notifications@github.com> escreveu:
